cypress/integration/admin_portal/tests/test_code_management.spec.js

```diff
@@ -67,51 +69,44 @@ describe('landing page tests', function () {
   })
 })
 
-//   it('checks the request codes form contents', function () {
-//     const request_codes_url = 'https://portal.stage.edx.org/successfactors/admin/coupons/request'
-//     let remainingRedemptions 
-//     cy.server()
+  it('checks for the assignment and revoking of the coupons', function () {
+    const request_codes_url = 'https://portal.stage.edx.org/successfactors/admin/coupons/request'
+    let remainingRedemptions 
+    cy.server()
 
-//     cy.route('GET','**/47059/codes/?code_filter=unassigned**').as('results')
-//     landingPage.goToEnterprise('SuccessFactors')
-//     landingPage.openCodeManagement()
-//     codeManagementDashboard.getTableRow().click()
-//     cy.wait("@results")
+    cy.route('GET','**/47059/codes/?code_filter=unassigned**').as('results')
```

Comment:
use a variable here for code id even if currently you are using a hard coded value